### ISSUE TITLE
some confusion about code

### ISSUE THREAD

**Issue by sunyue11** [Wangt-CN/MTFN-RR-PyTorch-Code#4]
sorry，I want to ask if you use the re-ranking in your code, because the loss function seem the same as the ranking loss.and it seems like that you do not construct text-text fusion

**Comment by Wangt-CN** [comment]
Hi, sorry for the late reply. The re-ranking is exactly used and it's a kind of "an after treatment". As discussed in paper, the rerank is not a loss during training, but more like an algorithm running in just few seconds. In this code base the text-text fusion is not included because of my bandwidth. However it's very similar to the image-text fusion network but just change the input to text-text. Please refer to issue #1 for the detail about text-text fusion and reranking.
